test/unit/link_checker_test.rb

```diff
@@ -0,0 +1,94 @@
+require "test_helper"
+require "gds_api/test_helpers/link_checker_api"
+
+class LinkCheckerTest < ActiveSupport::TestCase
+  include GdsApi::TestHelpers::LinkCheckerApi
+
+  def url
+    @url ||= "http://example.com"
+  end
+
+  def text
+    %(There is a link <a href="#{url}">here</a>)
+  end
+
+  def data
+    @data ||= text
+  end
+
+  def link_checker
+    @link_checker ||= LinkChecker.new(data)
+  end
+
+  def fixtures_path
+    Rails.root.join "test/fixtures/smart_answer_flows/flow-with-links"
+  end
```

Comment:
Can we put these in [test setup method](https://api.rubyonrails.org/classes/ActiveSupport/Testing/SetupAndTeardown.html)?

Reply:
We could - but it's a lot less flexible to do that.